src/main/fc/config.c

```diff
@@ -108,6 +108,7 @@ PG_RESET_TEMPLATE(systemConfig_t, systemConfig,
     .current_profile_index = 0,
     .current_battery_profile_index = 0,
     .debug_mode = SETTING_DEBUG_MODE_DEFAULT,
+    .osdNotifySettingsSaved = 0,
```

Comment:
@MrD-RC why this is part of systemConfig? This should not be part of systemConfig. The way I see, this is just a state, not configuration. Why do we want to persist `osdNotifySettingsSaved` at all? I know this code does not persists it, but it's possible.

Reply:
I'm going to take this part out, an put it in a different PR. I think the save could work more efficiently if a flag is set, then the save performed on the next cycle. It would avoid multiple save calls in the same loop.

The reason I put it in the config is that the save functions are part of the config. So it seemed the right place for the management of that state to be. I guess it could also be owned by the OSD.

Reply:
Let's remove it from this code and resolve separately. That would be nice